linker/Mono.Linker/Driver.cs

```diff
@@ -172,7 +172,7 @@ void Run ()
 			p.Process (context);
 		}
 
-		static void AddCustomStep (Pipeline pipeline, string arg)
+		protected void AddCustomStep (Pipeline pipeline, string arg)
```

Comment:
Side effect of some other changes.  Looks like I can shift some things around and change this back to static.

I will add back static